packages/webdriverio/src/commands/element/dragAndDrop.ts

```diff
@@ -122,10 +106,10 @@ export default async function dragAndDrop (
         id: 'finger1',
         parameters: { pointerType: 'mouse' },
         actions: [
-            { type: 'pointerMove', duration: 0, x: sourceX, y: sourceY },
+            { type: 'pointerMove', duration: 0, origin: this, x: 0, y: 0 },
             { type: 'pointerDown', button: ACTION_BUTTON },
             { type: 'pause', duration: 10 }, // emulate human pause
-            { type: 'pointerMove', duration, origin: 'pointer', x: targetX, y: targetY },
+            { type: 'pointerMove', duration, origin, x: targetX, y: targetY },
```

Comment:
Let's pull out the element id from the scope and only pass this along. It's a bit less data to upload:

```js
import { ELEMENT_KEY } from '../../constants'
import type { ElementReference } from '@wdio/protocols'

const { [ELEMENT_KEY] }:ElementReference = this

//...

   { type: 'pointerMove', duration: 0, origin: { [ELEMENT_KEY] }, x: 0, y: 0 },
```

Reply:
That's awesome, will do!